debian/patches/0001-disable-privileged-actions-test.patch

```diff
@@ -0,0 +1,22 @@
+Subject: Skip privileged actions test
```

Comment:
Do these patches need to be removed? Not sure how that works with a native package.